src/vs/platform/imageResize/electron-main/imageResizeMainService.ts

```diff
@@ -0,0 +1,70 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IImageResizeService } from '../common/imageResizeService.js';
+import { nativeImage } from 'electron';
+
+export class ImageResizeMainService implements IImageResizeService {
```

Comment:
Curious why we're doing this in electron sandbox rather than the browser as we already have a `resizeImage` implementation that runs in the browser.

If we implement it like this we'll still need an additional implementation in the browser layer such that MCP works in vscode.dev